src/uiTest/resources/project.rhino.js

```diff
@@ -380,7 +380,7 @@ function Project(underlyingProject) {
   };
 
   this.checkoutBranch = function (branchName) {
-    invokeActionAndWait('GitMachete.CheckoutSelectedBranchAction', ACTION_PLACE_CONTEXT_MENU, { SELECTED_BRANCH_NAME: branchName });
+    invokeActionAndWait('GitMachete.CheckoutSelectedAction', ACTION_PLACE_CONTEXT_MENU, { SELECTED_BRANCH_NAME: branchName });
```

Comment:
Wow so that's the only action that's explicitly called from UI tests 😯

Reply:
yup, and I think it's reasonable. we could use context menu action instead but this method is called so frequently (within the tests) so I think it would significantly extend the time of the tests run